openedx/core/lib/api/tests/test_authentication.py

```diff
@@ -45,7 +45,6 @@ def put(self, _request):
 
 
 urlpatterns = [
-    url(r'^oauth2/', include(('provider.oauth2.urls', 'oauth2'), namespace='oauth2')),
```

Comment:
**Note to Reviewer:** We found this issue when testing outside of Jenkins, because Jenkins had outdated cached dependencies.

Reply:
I believe there's other tests that will need to be fixed in this file, but this will at least allow tests to run.